### ISSUE TITLE
Issue with Decrypting with Transposition Cipher

### ISSUE THREAD

**Issue by galzandani** [jbainbridge98/cipherApp#2]
I run the code and try encrypting the key COMPUTERSECURITY with Transposition Cipher, which worked well. Then when I tried to decrypt my cipher text CERORIMSTPEYUCTU I came across the following error.

<img width="1440" alt="Screen Shot 2021-12-05 at 12 14 50 PM" src="https://user-images.githubusercontent.com/71564971/144756437-912553c1-aac5-4dab-a05a-31bacecc805b.png">

**Comment by jbainbridge98** [comment]
Yes, our transposition cipher seems to have some issues due to a global variable mishap.

**Comment by rkakadia9855** [comment]
Yeah it's a simple fix that we had already found after the code freeze. Just need to add the global keyword for the variable named generatedPlainText inside the function that decrypts Transposition cipher.

**Comment by symxmyz233** [comment]
It is **a normal bug**.
